Performance-Tracing: describe the process-tracing tool

elibarzilay opened the pull request.

amcasey reviewed (approved):
Three suggestions:
1) Mention that node 15 is required (unless that has changed)
2) Suggest min-duration as an alternative and solicit feedback
3) Provide an example command line

elibarzilay commented:
1. Fixed the `stream/promises` (use `promisify()` directly instead).
2. Added a suggestion but with a parenthesized comment saying that it might be better to sample.  Maybe you'll want to drop that to be more encouraging?
3. Added example.
4. Added more text to the [script's page](https://www.npmjs.com/package/process-tracing), which maybe you'll want to go over too?

amcasey commented:
On second thought, this is probably the wrong forum for requesting feedback on duration-filtering vs sampling, so I'd just drop everything about min-duration and provide a single, straightforward path through the instructions.

The npm readme looks good - thanks!
